packages/cli/test/integration.js

```diff
@@ -3277,6 +3277,33 @@ test('deploy gatsby twice and print cached directories', async t => {
   }
 });
 
+test('deploy pnpm twice using pnp and symlink=false', async t => {
+  const directory = path.join(__dirname, 'fixtures/unit/pnpm-pnp-symlink');
+
+  function deploy() {
+    return execa(binaryPath, [
+      directory,
+      ...defaultArgs,
+      '--public',
+      '--confirm',
+    ]);
+  }
+
+  function logs(deploymentUrl) {
+    return execa(binaryPath, ['logs', deploymentUrl, ...defaultArgs]);
+  }
+
+  let { stdout: deploymentUrl } = await deploy();
+  let { stdout: logsOutput } = await logs(deploymentUrl);
+
+  t.truthy(logsOutput.includes('No Build Cache available'));
```

Comment:
```suggestion
  t.regex(logsOutput, /No Build Cache available/m);
```

Lets use regex so when this test fails it will print the mismatch